docs/src/citation.md

```diff
@@ -0,0 +1,12 @@
+# Citation
+
+If your work uses this Element, please cite the following manuscript and Research
```

Comment:
Tried to request this as a change for you to commit, but it just went ahead and committed. `this Element` is what we've used in other repos rather than `element-name Element`

Reply:
Ok. Please avoid committing to my fork in the future unless we are jointly editing. Doing so muddies the water of Editor vs. Reviewer boundary and therefore the 'no-self-merge' policy

Reply:
Yep, figured out the use of `add a suggestion` tag on reviews